oop/homework.py

```diff
@@ -39,31 +39,61 @@ class Cat:
     """
 
     def __init__(self, age):
-        pass
+        self.age = age
+        self.saturation_level = 50
+        self._set_average_speed()
 
     def eat(self, product):
-        pass
+        if product == 'fodder':
```

Comment:
define dict with key-product, value-level and just call `self.increase_satureation_level(self.dict.get(product))`
so you will not longer need to set conditions